### PR TITLE
Update changelog for release-v1.5.0

### PR DESCRIPTION
**Pull request by csmbot**
Update changelog to include features and bug fixes

### CODE REVIEW

**Review by shaynafinocchiaro — APPROVED** [review]

File: CHANGELOG/CHANGELOG-1.5.0.md [hunk 1]
```diff
@@ -48,3 +67,5 @@
 - CSI-Powerstore: Update the upgrade procedure. ([#484](https://github.com/dell/csm/issues/484))
 - step_error: command not found in karavi-observability-install.sh. ([#479](https://github.com/dell/csm/issues/479))
 - "repctl cluster inject --use-sa" doesn't work for Kubernetes 1.24 and above. ([#463](https://github.com/dell/csm/issues/463))
+- Update CSM Observability release note with github issue link and support matrix. ([#455](https://github.com/dell/csm/issues/455))
```
Comment (baoy1) [inline CHANGELOG/CHANGELOG-1.5.0.md:70]: #455 if for v1.4.0 rather than v1.5.0. I removed the tag v1.5.0 from #455. 
FYI, @forrestxia .

Comment (baoy1) [reply CHANGELOG/CHANGELOG-1.5.0.md:70]: Hi @shaynafinocchiaro , can we delete line #70? `Update CSM Observability release note with github issue link and support matrix. ([#455](https://github.com/dell/csm/issues/455))`

Comment (YianZong) [reply CHANGELOG/CHANGELOG-1.5.0.md:70]: I don't see https://github.com/dell/csm/issues/519 in the bug list

Comment (shaynafinocchiaro) [reply CHANGELOG/CHANGELOG-1.5.0.md]: Hi @baoy1, the bug has been removed.

File: CHANGELOG/CHANGELOG-1.5.0.md [hunk 2]
```diff
@@ -6,31 +6,50 @@
...
+- CSM PowerFlex : QoS parameters for throttling performance and bandwidth of a CSI driver. ([#533](https://github.com/dell/csm/issues/533))
+- Standardize csi drivers helm installation. ([#494](https://github.com/dell/csm/issues/494))
+- CSM 1.5 release specific changes. ([#491](https://github.com/dell/csm/issues/491))
+- Scheduled Backups for Application Mobility. ([#551](https://github.com/dell/csm/issues/551))
+- SLES SP4 Support for CSI PowerFlex driver in CSM. ([#539](https://github.com/dell/csm/issues/539))
+- CSI-PowerScale: Add an option to the CSI driver force the client list to be updated even if there are unresolvable host. ([#534](https://github.com/dell/csm/issues/534))
+- FC connectivity with virtualized environment. ([#528](https://github.com/dell/csm/issues/528))
 - CSM Operator Supports Install of Authorization Proxy Server. ([#511](https://github.com/dell/csm/issues/511))
 - CSI-PowerScale: Add support for Standalone Helm charts. ([#506](https://github.com/dell/csm/issues/506))
+- CSM Operator: Support installation of PowerScale metrics for Observability module. ([#505](https://github.com/dell/csm/issues/505))
```
Comment (YianZong) [inline CHANGELOG/CHANGELOG-1.5.0.md:21]: Please remove #505, which is a duplicate to #488.

File: CHANGELOG/CHANGELOG-1.5.0.md (hunk 1 above)
Comment (shaynafinocchiaro) [reply CHANGELOG/CHANGELOG-1.5.0.md]: Hi @YianZong, 519 is not visible in the PR but I do see it under [Bugs](https://github.com/dell/csm/blob/changelog-staging/CHANGELOG/CHANGELOG-1.5.0.md#bugs)

**Comment by sharmilarama** [comment]
This bug [#164](https://github.com/dell/csm/issues/164) needs to be included.

File: CHANGELOG/CHANGELOG-1.5.0.md (hunk 1 above)
Comment (YianZong) [reply CHANGELOG/CHANGELOG-1.5.0.md]: Ok, I see, this is an incremental. Thank you, @shaynafinocchiaro

**Review by baoy1 — APPROVED** [review]

**Comment by shaynafinocchiaro** [comment]
Multiple Features and Bugs did not get included in this PR. I will merge this PR and open a new one.

File: CHANGELOG/CHANGELOG-1.5.0.md (hunk 2 above)
Comment (shaynafinocchiaro) [reply CHANGELOG/CHANGELOG-1.5.0.md:21]: This will be reflected in the next PR

**Pull request merged**